CPS-0020? | Governance Stakeholder Incentivization (#997)
cardano-foundation/CIPs: CPS-0019/README.md

---

> CPS: 19

Reviewer: ```suggestion
CPS: 20
```

> CPS: 0019

Reviewer 2: ```suggestion
CPS: 19
```
remove leading `0`s

> Title: Governance Stakeholder Incentivization  

Author: > I feel that incentization maybe _more_ the solution to the problem described here rather than the problem itself
> 
> maybe a adjustment to the title?

I would argue that depends on the scope we want to have with this CPS. Participation can include other aspects like education, user experience or culture. Participation is an important aspect, but not the Problem we are focusing on.

Status: Open  
Category: Tools  
Authors:  
- Seomon (Cardano After Dark) <Seomonregister@gmail.com>  
- Martin Marinov <martin.marinov@cardano.marketing>  
- Sebastián Pereira Gutiérrez <sebpereira33@gmail.com> 

Proposed Solutions:
- 

Discussions:  
- https://github.com/cardano-foundation/CIPs/pull/997

Created: 2025-02-13  
License: CC-BY-4.0

---

## Abstract

> The Cardano governance framework depends on the active participation of Delegated Representatives (DReps), the Constitutional Committee (CC), and Ada holders, alongside the already incentivized Stake Pool Operators (SPOs), maintain a highly effective, representative and democratic governance structure. Presently, the system's heavy reliance on the altruistic behavior of governance participants, which poses a significant risk, potentially leading to a concentration of voting power among a small, select group. This situation undermines broader community participation and threatens democratic principles. The primary goal should be to establish a balanced but robust incentivization framework that encourages consistent and informed participation from a diverse range of stakeholders with different levels of governance resposibilites, commitments and impact. However, achieving this objective faces several technical obstacles and game theoretic challenges, including designing fair reward mechanisms, ensuring the sustainability of incentives, and preventing the misuse of incentivization structures. Addressing these challenges is crucial to fostering a more balanced and representative governance model for the Cardano community.

Reviewer 3: ```suggestion
The Cardano governance framework depends on the active participation of Delegated Representatives (DReps), the Constitutional Committee (CC), and Ada holders, alongside the already incentivized Stake Pool Operators (SPOs), to maintain a highly effective, representative and democratic governance structure. Presently, the system's heavy reliance on the altruistic behavior of governance participants, which poses a significant risk, potentially leading to a concentration of voting power among a small, select group. This situation undermines broader community participation and threatens democratic principles. The primary goal should be to establish a balanced but robust incentivization framework that encourages consistent and informed participation from a diverse range of stakeholders with different levels of governance responsibilities, commitments and impact. However, achieving this objective faces several technical obstacles and game theoretic challenges, including designing fair reward mechanisms, ensuring the sustainability of incentives, and preventing the misuse of incentivization structures. Addressing these challenges is crucial to fostering a more balanced and representative governance model for the Cardano community.
```

## **Problem**

> The Cardano governance framework relies on Delegated Representatives (DReps), Constitutional Committee (CC), Stake Pool Operators (SPOs), and ADA holders to make informed decisions, exercise veto power, and select both SPOs and DReps that reflect the broader community's interests. However, the current system lacks a structured mechanism to incentivize DReps, CC members, and ADA holders delegating to DReps effectively. Without adequate rewards, DReps and CC members are primarily driven by altruism or selfinterest, which is neither sustainable nor scalable in the long term. ADA holders also lack any reason to deeply connect with DReps. This reliance on voluntary participation risks a significant imbalance in governance, where only a small, motivated subset of the community holds disproportionate influence.

Reviewer 3: ```suggestion
The Cardano governance framework relies on Delegated Representatives (DReps), Constitutional Committee (CC), Stake Pool Operators (SPOs), and ADA holders to make informed decisions, exercise veto power, and select both SPOs and DReps that reflect the broader community's interests. However, the current system lacks a structured mechanism to incentivize DReps, CC members, and ADA holders delegating to DReps effectively. Without adequate rewards, DReps and CC members are primarily driven by altruism or self-interest, which is neither sustainable nor scalable in the long term. ADA holders also lack any reason to deeply connect with DReps. This reliance on voluntary participation risks a significant imbalance in governance, where only a small, motivated subset of the community holds disproportionate influence.
```

The absence of an incentivization framework discourages widespread participation and may lead to voter apathy, reducing the overall quality and representativeness of decisions. This problem is further exacerbated by the potential for centralization, as a few highly active stakeholders can dominate governance processes, marginalizing the voices of the broader community.

The writing of this CPS document is motivated by the urgent need to address these issues and create a comprehensive incentivization system. Such a system aims to ensure that DReps, CC members, and ADA holders are fairly compensated for their efforts, thereby promoting active, informed participation and maintaining a decentralized governance structure that truly reflects the diverse interests of the Cardano community.

> Any incentives program for the above mentioned stakeholders should take into account the current SPO rewards. Rewards and incentives have to be considred along the yearly inflows of ADA to the treasury. All governance stakeholders must be considered as part of a whole which is the Cardano governance model. 

Reviewer 2: ```suggestion
Any incentives program for the above mentioned stakeholders should take into account the current SPO rewards. Rewards and incentives have to be considered along the yearly inflows of ada to the treasury. All governance stakeholders must be considered as part of a whole which is the Cardano governance model. 
```
spelling and grammar on ada

> 

Reviewer 2: Rather than just focusing on incentization,
is it applicable to discuss disincentives?
i.e. punishing stakeholders for not participating

Reviewer 2: such as the existing blocking of stake reward withdrawal for stake accounts not delegated to DRep

Author: Absolutely, disincentives, penalties or punishment can be used to prevent specific behaviour and define boundaries. The current solution of blocking stake reward withdrawal for stake accounts is not a great **longterm** solution but it got things started. Overall we should be careful with disincentives, but we also should mention them.
Let me see what we can do.

#### Maximize governance effectiveness
Traditional governance mechanisms often suffer from inefficiencies, slow decision-making, and resource-heavy processes that do not scale well. The incentivization framework for DReps must be designed to optimize governance effectiveness while minimizing unnecessary complexity and resource consumption.

## **Open Questions**
- Is change needed at the Ledger/protocol level or tooling/wallet level?
- What are the current incentives for stakeholders?
- Is the current architecture avoiding or aiding centralization?
- What’s the proper reward amount?
- How much should the rewards be based on ADA delegation, the DRep’s participation or other factors?
- What’s the impact of stakeholder rewards on Cardano’s treasury?

## **References**
- [Reward Schemes and Committee Sizes in Proof of Stake Governance](https://arxiv.org/abs/2406.10525)

> - [Should DReps receive compensation? Poll]()

Reviewer 2: missing link :)
